fdpa-resolutions.html

```diff
@@ -2,79 +2,21 @@
 <html>
   <head>
   <title>Fair Districts PA - Local Resolutions</title>
+  <meta name="referrer" content="origin">
   <style>
     #map-canvas { width:1100px; height:650px; }
-    .layer-wizard-search-label { font-family: sans-serif };
+    .layer-wizard-search-label { font-family: sans-serif }

```

Comment:
Semicolon after a style rule is incorrect syntax. Had to remove it to add another rule.